src/tools/singlejar/BUILD

```diff
@@ -228,9 +223,10 @@ cc_test(
     srcs = [
         "output_jar_simple_test.cc",
     ],
-    # TODO(@rongjiecomputer): update copts to handle Windows and add
-    # ".exe" extension.
-    copts = ["-DJAR_TOOL_PATH=\\\"$(JAVABASE)/bin/jar\\\""],
+    copts = select({
+        "//src/conditions:windows": ["-DJAR_TOOL_PATH=\\\"io_bazel/$(JAVABASE)/bin/jar.exe\\\""],
```

Comment:
What's the reason to add `io_bazel`?

Reply:
I am using runfiles library to find `jar.exe` (explained in [0]), and the runfiles manifest file have `io_bazel` prefix for the `jar.exe` entry.

[0]: https://github.com/bazelbuild/bazel/pull/6251#issuecomment-433294777

Reply:
Makes sense, thanks!